bioptim/misc/enums.py

```diff
@@ -126,3 +126,9 @@ class ConstraintType(Enum):
     USER = "user"
     INTERNAL = "internal"
     IMPLICIT = "implicit"
+
+
+class MultiBodyDynamics(Enum):
```

Comment:
Please call this class Dynamics (or something more generic than MULTI_BODY) as suggested in `bioptim/examples/torque_driven_ocp/example_soft_contact.py` Dynamics is probably already taken, but I let you figure out